tests/std/tests/P0009R18_mdspan_layout_left/test.cpp

```diff
@@ -75,14 +75,16 @@ constexpr void do_check_members(const extents<IndexType, Extents...>& ext, index
         // Other tests are defined in 'check_construction_from_other_right_mapping' function
     }
 
-#pragma warning(push) // TRANSITION, "/analyze:only" BUG?
-#pragma warning(disable : 28020) // The expression '0<=_Param_(1)&&_Param_(1)<=1-1' is not true at this call
     { // Check construction from layout_stride::mapping
         array<IndexType, Ext::rank()> strides{};
         if constexpr (Ext::rank() > 0) {
             strides.front() = 1;
             for (size_t i = 1; i < Ext::rank(); ++i) {
-                strides[i] = static_cast<IndexType>(strides[i - 1] * ext.extent(i - 1));
+#pragma warning(push)
+#pragma warning(disable : 28020) // TRANSITION, DevCom-923103
+                strides[i] =
+#pragma warning(pop)

```

Comment:
It seems really weird to scope this to a subexpression instead of a full expression, even if that's possible. Although this is just a test and it's compiling fine right now, I think I would prefer to wrap the whole statement (possibly this comes from paranoia about "weird code" triggering compiler bugs).